.github/workflows/build-release-binary.yml

```diff
@@ -164,7 +164,7 @@ jobs:
       contents: read
       packages: write
     env:
-      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      GITHUB_TOKEN: ${{ secrets.BOTTY_GITHUB_TOKEN }}
```

Comment:
If the user is not botty, we should name this something else!

Reply:
Done, renamed all to `ITCHY_`